lib/src/util.dart

```diff
@@ -15,12 +15,18 @@
 import 'dart:math' as math;
 import 'dart:io';
 
+import 'package:codemod/codemod.dart';
 import 'package:io/ansi.dart';
 import 'package:source_span/source_span.dart';
 
 import 'constants.dart';
 import 'patch.dart';
 
+Future<String> applySuggestor(FileContext context, Suggestor suggestor) async {
+  final patches = await suggestor.generatePatches(context).toList();
+  return applyPatches(context.sourceFile, patches);
+}
```

Comment:
I found myself wanting this in unit tests, so I added it. Makes the unit tests just a bit more terse.